pkg/helper/machinepools/helpers_test.go

```diff
@@ -187,6 +187,35 @@ var _ = Describe("Create node drain grace period builder validations", func() {
 			"1 minute",
 			false,
 		),
+		Entry("Should not error with hour unit",
+			"1 hour",
+			false,
+		),
+	)
+})
+
+var _ = Describe("Validate node drain grace period", func() {
+	DescribeTable("Validate node drain grace period",
+		func(period interface{}, hasError bool) {
+			err := ValidateNodeDrainGracePeriod(period)
+			if hasError {
```

Comment:
@philipwu08 Could you update this to verify that the error is the one you expect. You could consider passing in the expected error message / substring and check that it matches.

Could you also add a couple more test cases to check the untested paths mentioned here: https://app.codecov.io/gh/openshift/rosa/pull/1866?dropdown=coverage&src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=checks&utm_campaign=pr+comments&utm_term=openshift

We should be able to get to 100% coverage on the `ValidateNodeDrainGracePeriod` function